src/test/java/com/training/assignment/tests/AssignmentTest.java

```diff
@@ -116,4 +123,22 @@ public void testOperationWithLoops() {
                     .as("Number is not an odd number-->" + oddNumber).isTrue();
         }
     }
+
+    @Test
+    public void testOopsConcept() {
+        List<String> sportsList = new ArrayList<>(Arrays.asList("Cricket", "Khokho", "Kabaddi", "Batminton", "Hockey", "Football"));
+
+        softAssert.assertThat(operation.getSportsName(sportsList, "Batminton"))
+                .as("List should not contains Batminton").isEqualTo(Arrays.asList("Cricket", "Khokho", "Kabaddi", "Hockey", "Football"));
+
+        String firstName = "MIKE";
+        String surname = "TYSON";
+
+        softAssert.assertThat(parentChild.getName()).as("Full name  of parent child is incorrect").isEqualTo(firstName + " " + surname);
+
+        String actorWork = "Acting";
+        String directorWork = "Directing";
+
+        softAssert.assertThat(director.getWorkDetail()).as("Work of actor and director differs").isEqualTo(actorWork + " " + directorWork);
```

Comment:
softAssert.assertThat(director.getWorkDetail()).as("Work of actor and director differs").isEqualTo("Acting Directing");